b2share/modules/communities/cli.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2016 University of Tuebingen, CERN.
+# Copyright (C) 2015 University of Tuebingen.
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""B2Share Communities module Command Line Interface."""
+
+from __future__ import absolute_import
+
+from os.path import isfile
+
+import click
+from flask_cli import with_appcontext
+
+from invenio_db import db
+
+from .api import Community, CommunityDoesNotExistError
+
+#from .errors import RootSchemaAlreadyExistsError
+#from .helpers import load_root_schemas
+
+@click.group()
+def communities():
+    """communities management commands."""
+
+
+@communities.command()
+@with_appcontext
+@click.option('-v', '--verbose', is_flag=True, default=False)
+@click.argument('name')
+@click.argument('description')
+@click.argument('logo')
+def create(verbose, name, description, logo):
+    """Creating a community in the database. Name can be 255 characters long. Description is a text of maximally 1024 characters enclosed in parentheses. The logo parameter should be a valid path to a logo file. """
```

Comment:
s/Creating/Create/g

In PEP-257: *The docstring is a phrase ending in a period. [It prescribes the function or method's effect as a command](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".*

Regarding the additional information, such as parameter types and description, [we use sphinx napoleon plugin](http://www.sphinx-doc.org/en/stable/ext/napoleon.html) which enables us to use [google style format for docstrings](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments)